sqlconnect/internal/redshift/driver/dsn.go

```diff
@@ -179,6 +214,14 @@ func (cfg *RedshiftConfig) setParams(params url.Values) error {
 		cfg.SessionToken = params.Get("sessionToken")
 		cfg.Params.Del("sessionToken")
 	}
+	if params.Has("roleARN") {
```

Comment:
Please add a test case in `TestRedshiftDataConfig__String`

Reply:
Added a test.